app/screens/settings/about/about.tsx

```diff
@@ -1,16 +1,18 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+import Clipboard from '@react-native-clipboard/clipboard';
 import React, {useCallback, useMemo} from 'react';
 import {useIntl} from 'react-intl';
 import {Alert, Text, View} from 'react-native';
 import DeviceInfo from 'react-native-device-info';
 
+import {buttonBackgroundStyle, buttonTextStyle} from '@app/utils/buttonStyles';
```

Comment:
```suggestion
import {buttonBackgroundStyle, buttonTextStyle} from '@utils/buttonStyles';
```

@larkox you may have missed this